config/default.json

```diff
@@ -293,6 +293,12 @@
       "endpoint": "http://localhost:8080/api/ext/cms-data/cms{{type}}/{{cmsId}}",
       "endpointIdentifier": "http://localhost:8080/api/ext/cms-data/cms{{type}}Identifier/{{cmsIdentifier}}/storeId/{{storeId}}"
     },
+    "cms_block": {
+      "max_count": 500
+    },
+    "cms_page": {
+      "max_count": 500
+    },
```

Comment:
We shouldn’t load all cms blocks and pages as it will cause sending them with the __INITIAL_STATE__ output and can be harmful boot h for performance + SEO

Reply:
this config option will need to easily manage  load of all blocks list if it needs as we need to have size provided for ES to get the data overwise it will be  only 10